app/Http/Controllers/Users/BulkUsersController.php

```diff
@@ -249,4 +251,38 @@ protected function logItemCheckinAndDelete($items, $itemType)
             $logAction->logaction('checkin from');
         }
     }
+    protected function findAssetOwners()
```

Comment:
I don't think this method is needed here. We have existing methods on the model that we can likely use instead of duplicating code here, esp since this method is only used once.